code/game/objects/effects/effect_system/effects_other.dm

```diff
@@ -46,16 +44,29 @@
 /datum/effect_system/trail_follow/generate_effect()
 	if(!check_conditions())
 		return stop()
-	if(oldposition && !(oldposition == get_turf(holder)))
-		if(!oldposition.has_gravity() || !nograv_required)
-			var/obj/effect/E = new effect_type(oldposition)
-			set_dir(E)
-			if(fade)
-				flick(fadetype, E)
-				E.icon_state = ""
-			if(qdel_in_time)
-				QDEL_IN(E, qdel_in_time)
-	oldposition = get_turf(holder)
+	if(cooldown < world.time)
+		if(!holder.has_gravity() || !nograv_required)
+			if(ismovableatom(holder))
+				var/atom/movable/AM = holder
+				for(var/turf/K in AM.locs)
+					effects_list |= new effect_type(K)
```

Comment:
creates a jetpack effect on every turf the jetpacker is on, looks a bit better than an off-center trail

Reply:
I would rather you have a single trail effect and offset it behind them